src/sentry/api/endpoints/user_authenticator_details.py

```diff
@@ -117,7 +127,7 @@ def delete(self, request, user, auth_id, interface_device_id=None):
                 context={
                     'authenticator': authenticator,
                 },
-                send_email=False
+                send_email=True
```

Comment:
Do you want to send more specific info here (i.e. which device was removed, etc.) or is the general mfa-removed email okay. Presumably, they'd know what action they just took

Reply:
It would be nice if we did have the device name, I think it'd be helpful in the case where they have been compromised.